### DISCUSSION TITLE
Can I relocate the dark-mode button

### DISCUSSION THREAD

**Discussion by josherrickson** [quarto-dev/quarto-cli#6891] [Q&A]
### Description

When generating a html formatted document, the dark-mode button appears in the top-right corner, disappearing as the user scrolls. Is there anyway to relocate that button - preferably into the TOC?

**Comment by mcanouil** [comment]
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Comment by josherrickson** [comment]
````qmd
---
title: "Reproducible Quarto Document"
format: 
  html:
    toc: true
    theme:
      light: flatly
      dark: darkly

---

# Section 1

<br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br />

# Section 2

<br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br />

# Section 3

<br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br />
```

**Comment by mcanouil** [comment]
Oh I see now.
Unfortunately, I don't think there is a way to move the button from within Quarto.
Using JavaScript it might be possible but not trivial at all.
